--- a/build.sh
+++ b/build.sh
@@ -98,7 +98,7 @@
          -c wasm-build/sdk_port-wasi/sdk_port-wasi-dlfcn.c \
          -Wno-incompatible-pointer-types
         then
-            COPTS="$LOPTS" ${CC} ${CC_PGLITE} -ferror-limit=1 -Wl,--no-stack-first -Wl,--global-base=${GLOBAL_BASE_B} -o ${PG_DIST}/pglite.wasi \
+            COPTS="$LOPTS" ${CC} ${CC_PGLITE} -ferror-limit=1 -Wl,--global-base=${GLOBAL_BASE_B} -o ${PG_DIST}/pglite.wasi \
              -nostartfiles ${PGINC} ${BUILD_PATH}/pglite.o \
              ${BUILD_PATH}/sdk_port-wasi.o \
              $LINKER $LIBPGCORE \
